packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -311,15 +311,18 @@ export async function runtimeLogging(command: commander.Command) {
   }
 }
 
-export async function sideload(manifestPath: string, command: commander.Command) {
+export async function sideload(manifestPath: string, platform: string, command: commander.Command) {
```

Comment:
As far as I can tell, this breaking change does not impact our existing project templates. I did testing of Office-Add-Taskpane with this change, and "npm start" and "npm run start web" still work

Reply:
Sure, but this is an exported function so code that is using it will be broken by adding the parameter in the middle. I strive to follow semantic versioning properly, which means we should just bump the major version of this when we release it. Unless we add it to the end, which would make it a non-breaking change.

Reply:
I personally would like for this to be a non-breaking change, but not sure if this is possible if we opt to make it a required param, which I think is the right thing to do.  Not sure I understand this comment, which would seem to suggest making it an optional param again:  "Unless we add it to the end, which would make it a non-breaking change"

Reply:
With JavaScript, any parameters not specified when a call is made have the value `undefined` so adding additional parameters won't break things. With TypeScript, since this could get a value undefined, it needs to be an optional param. Therefore, adding `platform?: string` on the end would be OK. However, I think commander may require the commander.Command argument to be at the end. I don't recall exactly. If it does require the `command` parameter at the end, then there's no way to make it a non-breaking change. However, as I look at this, since it is just the command implementation, and not the actual sideload() function from sideload.ts, I don't think it is as big of a deal to have this change since there's no reason to use this exported function from command.ts. This is a long-winded way to say that I now think it is OK to add this platform parameter here.